### PR TITLE
build: one version to rule them all

### PR DESCRIPTION
**Pull request by tomberek**
## Proposed Changes
Simpler versioning procedure

CLI will no longer include revCount when built from flox/flox, only for nightly releases.

## Release Notes
N/A

### CODE REVIEW

File: cli/flox/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 [package]
 name = "flox"
-version = "1.0.4"
```
Comment (ysndr) [inline cli/flox/Cargo.toml:3]: surprised that cargo will default to `0.0.0` if thre is no version field, i thought that was required.
anyway can we set the version to `0.0.0` explicitily like in the other crate(s)?

File: doc/release-notes/release-notes.md
Comment (ysndr) [inline doc/release-notes/release-notes.md:1]: where did them release noted go?

File: cli/flox/Cargo.toml (hunk 1 above)
Comment (garbas) [reply cli/flox/Cargo.toml]: `version` field is not mandatory. It is mandatory if you will publish the crate, which is not what we are doing.

Comment (garbas) [reply cli/flox/Cargo.toml]: I'll set the version to `0.0.0` which is anyway the default version.

**Review by ysndr — APPROVED** [review]

**Pull request merged**